src/main/resources/templates/add_sighting.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html layout:decorate="layout">
+    <head lang="en"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    xmlns:th="https://www.thymeleaf.org">
+    <link th:href="@{/css/addSighting.css}" rel="stylesheet" />
+    <link th:href="@{/css/application.css}" rel="stylesheet" />
+    </head>
+    <body>
+        <div layout:fragment="content">
+            <h1 id="add-sighting-header">Add a whale sighting</h1>
+            <p id="add-sighting-details">Please add the details of your sighting below.</p>
+            <form id="add-sighting" action="" method="POST">
+                <label for="date">
+                    Date of Sighting
+                </label>
+                <input type="date" required/>
+                <label for="latitude">
+                    Latitude
+                </label>
+                <input type="number" required placeholder="e.g. 38.8951"/>
```

Comment:
You've used the `for` field, but for this to work, you need to also set the ID of the input field to the same thing
```suggestion
                <label for="latitude">
                    Latitude
                </label>
                <input id="latitude" type="number" required placeholder="e.g. 38.8951"/>
```

Depending on how this wires up elsewhere, you may also need to add the `name` attribute to these input elements

Reply:
Please make this change on the other inputs too!